polkadot-parachains/statemine/src/weights/pallet_utility.rs

```diff
@@ -39,4 +39,7 @@ impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 			// Standard Error: 0
 			.saturating_add((4_578_000 as Weight).saturating_mul(c as Weight))
 	}
+	fn dispatch_as() -> Weight {
+		(14_340_000 as Weight)
```

Comment:
should we run benchmark on the PR ?
Anyhow I guess we'll run them before release

Reply:
also this kind of benchmark is unaffected by which runtime it is inside